### PR TITLE
Fix DFV field view on settings pages

### PR DESCRIPTION
**Pull request by nicdford**
Fix for #4518

### CODE REVIEW

**Comment by nicdford** [comment]
Hold off on merging this, i didn't fully test how this would impact other fields, so i'm going through that now.

File: ui/styles/src/base/_admin.scss [hunk 1]
```diff
@@ -486,9 +486,9 @@ div.pods-manage-field .pods-field-option-group .pods-field-option-group-label {
 /**
  * Pods DFV
  */
-.pods-manage-field .pods-dfv-field .pods-pick-values > ul {
-    border: 0;
-}
+// .pods-manage-field .pods-dfv-field .pods-pick-values > ul {
```
Comment (pglewis) [inline ui/styles/src/base/_admin.scss]: We have a history in git, best to just remove lines unless there's another compelling reason here

Comment (nicdford) [reply ui/styles/src/base/_admin.scss]: Affirmative. Shaking off the rust :)

**Comment by nicdford** [comment]
OK, this should be a much better fix IMO. Let me know if something doesn't look right and I can fix it.

**Comment by JoryHogeveen** [comment]
@nicdford 
Can you post a screenshot of what it should look like with this fix? I think I'm missing something.

File: ui/styles/src/base/_admin.scss [hunk 2]
```diff
@@ -486,8 +479,9 @@ div.pods-manage-field .pods-field-option-group .pods-field-option-group-label {
 /**
  * Pods DFV
  */
-.pods-manage-field .pods-dfv-field .pods-pick-values > ul {
-    border: 0;
+.pods-manage-field .pods-dfv-field .pods-ui-list-autocomplete {
```
Comment (JoryHogeveen) [inline ui/styles/src/base/_admin.scss]: The images view should be similar right?
`.pods-manage-field .pods-dfv-field .pods-ui-file-list.pods-field-template-rows`.

**Comment by pglewis** [comment]
Is this one ready to go now in your opinion, @JoryHogeveen ?

**Comment by JoryHogeveen** [comment]
Yep, works for me. Might have some improvements but that will require a larger refactor so let's punt that to a later version.

**Pull request merged**